src/dispatch.c

```diff
@@ -51,7 +51,7 @@ SEXP method_internal(SEXP table, SEXP signature, R_xlen_t signature_itr, SEXP ig
 SEXP get_class(SEXP object, SEXP envir) {
     static SEXP fun = NULL;
     if (fun == NULL) {
-      fun = Rf_findVarInFrame(R_BaseEnv, Rf_install("class"));
+      fun = Rf_findVarInFrame(R_BaseEnv, Rf_install(".class2"));
```

Comment:
This is kind of nice for matrices/arrays too, even though I think we haven't fully thought about those yet

> .class2(matrix(1))
[1] "matrix"  "array"   "double"  "numeric"
> class(matrix(1))
[1] "matrix" "array"

Reply:
Yeah, we could always back port, if needed.